### PR TITLE
Fixing issue 2466

### PR DESCRIPTION
**Pull request by johnpuddephatt**
See #2466.

### CODE REVIEW

**Comment by brendo** [comment]
Hey @johnpuddephatt, can you open this against `2.6.x` instead of `integration` please?

File: symphony/assets/js/src/symphony.notify.js [hunk 1]
```diff
@@ -69,6 +70,13 @@
 				storage = (window.localStorage[settings.storage]) ? $.parseJSON(window.localStorage[settings.storage]) : [];
 			}
 
+			// Dimmed success transition
+			if (!!type && !!~type.indexOf('success')) {
```
Comment (brendo) [inline symphony/assets/js/src/symphony.notify.js]: I haven't see this syntax before, what does `!!~` do? Out of curiosity :)

**Comment by brendo** [comment]
Patch manually applied due to merge conflicts, thanks @johnpuddephatt!

File: symphony/assets/js/src/symphony.notify.js (hunk 1 above)
Comment (michael-e) [inline symphony/assets/js/src/symphony.notify.js]: Seems that it's equivalent to `ToBoolean`, see: http://stackoverflow.com/questions/10467475/double-negation-in-javascript-what-is-the-purpose

**Pull request closed without merging**